src/main/java/seatsio/events/Events.java

```diff
@@ -268,11 +269,19 @@ public ChangeObjectStatusResult release(String eventKey, List<?> objects, String
     }
 
     public ChangeObjectStatusResult release(String eventKey, List<?> objects, String holdToken, String orderId, Boolean keepExtraData, Boolean ignoreChannels, Set<String> channelKeys) {
-        return changeObjectStatus(eventKey, objects, FREE, holdToken, orderId, keepExtraData, ignoreChannels, channelKeys);
+        return releaseObjectStatus(singletonList(eventKey), objects, holdToken, orderId, keepExtraData, ignoreChannels, channelKeys);
     }
 
     public ChangeObjectStatusResult release(List<String> eventKeys, List<?> objects, String holdToken, String orderId, Boolean keepExtraData, Boolean ignoreChannels, Set<String> channelKeys) {
-        return changeObjectStatus(eventKeys, objects, FREE, holdToken, orderId, keepExtraData, ignoreChannels, channelKeys);
+        return releaseObjectStatus(eventKeys, objects, holdToken, orderId, keepExtraData, ignoreChannels, channelKeys);
+    }
+
+    private ChangeObjectStatusResult releaseObjectStatus(List<String> eventKeys, List<?> objects, String holdToken, String orderId, Boolean keepExtraData, Boolean ignoreChannels, Set<String> channelKeys) {
```

Comment:
Maybe `releaseObject` would be a better name? It's not clear to me what `releaseObjectStatus` means.

Reply:
👍